### ISSUE TITLE
[BUG] Replica hang without progress sometimes

### ISSUE THREAD

**Issue by yasker** [longhorn/longhorn#1979]
**Describe the bug**
In v1.0.2, sometimes the replica hangs when attaching the volume. It seems the replica instance manager is locked up.

Probably a duplicate of https://github.com/longhorn/longhorn/issues/1599 but need to confirm.

**To Reproduce**
Unknown. Seems related to the node reboot.

**Expected behavior**
The volume should be attached.

**Log**
Not available for now.

**Environment:**
 - Longhorn version: v1.0.2
 - Kubernetes version: unknown
 - Node OS type and version: unknown.

**Additional context**
Screenshot
![image (12)](https://user-images.githubusercontent.com/3344618/98972574-754cee80-24c7-11eb-95ea-572e08bf2978.png)

**Comment by mattmattox** [comment]
Lab Cluster logs: [longhorn-support-bundle_7a2e9979-e897-11e9-9dbd-005056a259f5_2020-11-12T15-14-48Z.zip](https://github.com/longhorn/longhorn/files/5532759/longhorn-support-bundle_7a2e9979-e897-11e9-9dbd-005056a259f5_2020-11-12T15-14-48Z.zip)

Also, it looks like /var/lib/longhorn wasn't formatted from the node in question. So /var/lib/longhorn was returning I/O errors. After fixing the filesystem, I recycled the stuck manager but the node required a reboot to fully resolve.

**Comment by shuo-wu** [comment]
The symptom/log in the support bundle is exactly the same as the #1599. It's a duplicate issue.
```
2020-11-12T04:42:58.371931295Z time="2020-11-12T04:42:58Z" level=debug msg="Instance handler updated instance pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123 state, old state , new state stopped"
2020-11-12T04:44:02.177921224Z time="2020-11-12T04:44:02Z" level=debug msg="Prepare to create instance pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123"
2020-11-12T04:47:02.177986152Z E1112 04:47:02.177683       1 replica_controller.go:177] fail to sync replica for longhorn-system/pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123: failed to start process: rpc error: code = DeadlineExceeded desc = context deadline exceeded
2020-11-12T04:47:02.178220598Z time="2020-11-12T04:47:02Z" level=warning msg="Dropping Longhorn replica longhorn-system/pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123 out of the queue: fail to sync replica for longhorn-system/pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123: failed to start process: rpc error: code = DeadlineExceeded desc = context deadline exceeded"
2020-11-12T04:47:02.178250372Z time="2020-11-12T04:47:02Z" level=info msg="Event(v1.ObjectReference{Kind:\"Replica\", Namespace:\"longhorn-system\", Name:\"pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123\", UID:\"f8ce2870-29a8-49cd-a393-34b202951ef5\", APIVersion:\"longhorn.io/v1beta1\", ResourceVersion:\"147933831\", FieldPath:\"\"}): type: 'Warning' reason: 'FailedStarting' Error starting pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123: failed to start process: rpc error: code = DeadlineExceeded desc = context deadline exceeded"
2020-11-12T04:47:55.756082520Z time="2020-11-12T04:47:55Z" level=error msg="failed to poll instance info to update instance manager instance-manager-r-c509f7c2: failed to list processes: rpc error: code = DeadlineExceeded desc = context deadline exceeded"
```
```
2020-11-12T04:37:55.318553032Z time="2020-11-12T04:37:55Z" level=info msg="Storing process logs at path: /var/log/instances"
2020-11-12T04:37:55.326037110Z [longhorn-instance-manager] time="2020-11-12T04:37:55Z" level=info msg="Instance Manager listening to 0.0.0.0:8500"
2020-11-12T04:37:55.759616671Z [longhorn-instance-manager] time="2020-11-12T04:37:55Z" level=debug msg="started new process manager update watch"
2020-11-12T04:44:02.188091613Z [longhorn-instance-manager] time="2020-11-12T04:44:02Z" level=info msg="Process Manager: prepare to create process pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123"
2020-11-12T04:44:02.188143278Z [longhorn-instance-manager] time="2020-11-12T04:44:02Z" level=info msg="Process Manager: created process pvc-a4c25c79-f336-404d-96b8-66ae27ef3573-r-ef6ff123"
```

**Comment by yasker** [comment]
Thanks @shuo-wu .

Close as a duplicate of #1599, which will be fixed in v1.1.0.
